pegasus/sites.v3/code.org/views/critical_font.js.erb

```diff
@@ -14,13 +14,13 @@ if (supportsFontLoading) {
   var fontDataArray = new Uint8Array(fontData);
   var fontOptions = {
     style: 'normal',
-    weight: 'normal',
+    weight: '400',
     // 0-9,A-Z,a-z
     unicodeRange: 'U+0020,U+0030-0039,U+0041-005A,U+0061-007A'
   };
 
-  window.criticalFonts = ['4r', '5r', '7r', '4i', '5i', '7i'].map(function(path) {
-    var family = "Gotham " + path;
+  window.criticalFonts = [''].map(function() {
```

Comment:
nit: should we go ahead and remove the `map` here since there's only one font family?